packages/es-components/src/components/controls/answer-group/AnswerGroup.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const AnswerSet = styled.div`
+  display: flex;
+  flex-wrap: nowrap;
+
+  @media (min-width: ${props => props.theme.screenSize.desktop}) {
+  }
+`;
+
+function AnswerGroup({
+  name,
+  children,
+  itemWidth,
+  styleType,
+  selectedType,
+  ...rest
+}) {
+  const renderButtons = () =>
+    React.Children.map(children, (child, index) => {
+      const key = `${name}-option-${index + 1}`;
```

Comment:
I think this is fine for now, but we should be careful of deriving `key` from `index`. If the number of children is dynamic this method of key generation is problematic.